move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -61,8 +61,26 @@ void move_group::MoveGroupMoveAction::initialize()
 void move_group::MoveGroupMoveAction::executeMoveCallback(const moveit_msgs::MoveGroupGoalConstPtr& goal)
 {
   setMoveState(PLANNING);
-  context_->planning_scene_monitor_->updateFrameTransforms();
 
+  const ros::Time now = ros::Time::now();
+  for (int i = 0; ; ++i)
+  {
+    ros::Duration age = now - context_->planning_scene_monitor_->getLastRobotStateUpdateTime();
+    if (age < ros::Duration(0))
+      break;
+
+    if (i >= 200)
```

Comment:
@andreaskoepf: see https://github.com/ros-planning/moveit_ros/pull/686 for integrating changes to the planning scene synchronously.